Mathlib/SetTheory/Lists.lean

```diff
@@ -437,6 +437,10 @@ termination_by
 #align lists.subset.decidable Lists.Subset.decidable
 #align lists.mem.decidable Lists.mem.decidable
 
+-- This is otherwise detected by the `nonClassInstance` linter from
+-- https://github.com/leanprover/std4/pull/268
+attribute [-instance] Lists.Equiv.decidable._unary._mutual
```

Comment:
@alexjbest do you undestand what is happening here?

(Let's not wait for a resolution to this before merging this PR.)

Reply:
I think its simply that when you make a mutual `def/theorem/instance` lean creates the internal autogenerated mutual parts as `def/theorem/instance` correspondingly, which seems non-ideal in the `instance` cases but not high priority to change as mutual instances are quite rare.